metricbeat/module/mongodb/_meta/replicaset.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+echo "Waiting for startup.."
+until curl http://mongodb:27017/serverStatus\?text\=1 2>&1 | grep uptime | head -1; do
+  printf '.'
+  sleep 1
+done
+
+echo curl http://mongodb:27017/serverStatus\?text\=1 | grep uptime | head -1
+echo "Started.."
+
+sleep 10
+
+mongo --host mongodb:27017 <<EOF
+   var cfg = {
+        "_id": "rs",
+        "version": 1,
+        "members": [
```

Comment:
According to [mongodb documentation](https://docs.mongodb.com/manual/tutorial/deploy-shard-cluster/#create-the-shard-replica-sets), there can be _single-member replica sets_ for testing purpouses, this would simplify the testing of this metricset and I think it'd be enough by now.

Reply:
👍 Nice idea. I'll revert those cluster configuration and try to setup replicaset on the old MongoDB container.